README.md

```diff
@@ -39,13 +39,22 @@ Usage of ./mysql-tester:
         The user for connecting to the database. (default "root")
   -xunitfile string
         The xml file path to record testing results.
+  -check-error
+        If --error ERR does not match, return error instead of just warn
+  -suffix
```

Comment:
It's better to rename it to 'extension' now since 'file extension' is the formal term.